src/inspector/view/qml/MuseScore/Inspector/notation/notes/BeamSettings.qml

```diff
@@ -194,17 +194,25 @@ FocusableItem {
                     width: parent.width
 
                     FlatRadioButton {
+                        id: crossBeamUp
                         width: 0.5 * parent.width - 2
                         iconCode: IconCode.ARROW_UP
                         checked: root.model ? root.model.crossStaffMove.value < 0 : false
                         onClicked: root.model.crossStaffMove.value -= 1
+
+                        navigation.panel: root.navigation.panel
+                        navigation.row: crossStaffMove.navigationRowStart
```

Comment:
`crossStaffMove.navigationRowStart` is already taken by the "reset" button inside InspectorPropertyView. So you'll need to go for +1.